test/ably/resthttp_test.py

```diff
@@ -135,3 +139,16 @@ def test_custom_http_timeouts(self):
         self.assertEqual(ably.http.http_open_timeout, 8)
         self.assertEqual(ably.http.http_max_retry_count, 6)
         self.assertEqual(ably.http.http_max_retry_duration, 20)
+
+    def test_request_headers(self):
+        ably = AblyRest(key=test_vars["keys"][0]["key_str"],
+                        rest_host=test_vars["host"],
+                        port=test_vars["port"],
+                        tls_port=test_vars["tls_port"],
+                        tls=test_vars["tls"])
+        r = ably.http.make_request('HEAD', '/time', skip_auth=True)
+        self.assertIn('X-Ably-Version', r.request.headers)
+        self.assertEqual(r.request.headers['X-Ably-Version'], api_version)
+
+        self.assertIn('X-Ably-Lib', r.request.headers)
+        self.assertEqual(r.request.headers['X-Ably-Lib'], 'python-%s' % lib_version)
```

Comment:
As above, except perhaps use a regex i.e. `/1\.\0.\d+/` so that we know that all patch updates don't effect the tests, but a minor or major release change will break this test which is a good thing.